picard/formats/apev2.py

```diff
@@ -160,7 +160,9 @@ def _load(self, filename):
                     name = name_lower
                     if name == 'year':
                         name = 'date'
-                        value = sanitize_date(value)
+                        config = get_config()
```

Comment:
It would be much better to set `config` outside loops.
In this case, I would even move `config.setting['disable_date_sanitize']` outside the loops and cache the value.

Something like:

```python
        config = get_config()
        date_sanitize =  not config.setting['disable_date_sanitize']
        if file.tags:
            for origname, values in file.tags.items():
...
                            if date_sanitize:
                                value = sanitize_date(value) 
```